INSTALL.md

```diff
@@ -128,7 +130,7 @@ source env/bin/activate
 synctl start
 ```
 
-#### Platform-Specific Instructions
+#### Platform-Specific prerequisites
```

Comment:
Please either capitalise "prerequisites" or make the first letter of "specific" lowercase so that the title is consistent.